frontend/ui/data/strings.lua

```diff
@@ -77,8 +77,10 @@ S.HIGH = _("high")
 S.ZERO_DEG = _("0 deg")
 S.FIVE_DEG = _("5 deg")
 S.TEN_DEG = _("10 deg")
-S.PORTRAIT = _("portrait")
-S.LANDSCAPE = _("landscape")
+S.PORTRAIT = _("up")
+S.LANDSCAPE = _("right")
+S.PORTRAIT_ROTATED = _("down")
```

Comment:
In the same vein, I tend to refer to those as "Inverted", elsewhere in the code.

But it's only a variable name here, so, meh ;).

Reply:
I copied `ROTATED` from the framebuffer code and in the cleanup happened to get rid of all the inverted references, to me rotated or inverted are the same, but we should make sure this matches the FB code